meinberlin/react/livequestions/react_questions_statistics.jsx

```diff
@@ -1,13 +0,0 @@
-import StatisticsBox from './StatisticsBox'
```

Comment:
@vellip I noticed that `<StatisticsBox />` was also rendered in react_questions_statistics.jsx, but react_questions_statistics.jsx isn't used anywhere or referenced in webpack.common.js, so have removed it.